src/portable/portable.c

```diff
@@ -719,9 +719,9 @@ static int extract_image_and_extensions(
                         return r;
 
                 if (validate_extension) {
-                        r = extension_release_validate(ext->path, id, version_id, sysext_level, "portable", extension_release, IMAGE_SYSEXT);
+                        r = extension_release_validate(ext->path, id, /* host_os_release_id_like = */ NULL, version_id, sysext_level, "portable", extension_release, IMAGE_SYSEXT);
```

Comment:
please implement this here too - the call above can parse ID_LIKE, like in other places, so it should be simple to do

Reply:
done